src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import { SortType } from '../../types/SortType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  sortField: SortType;
+  setSortField: (field: SortType) => void;
+  todos: Todo[];
+};
+
+export const Footer: React.FC<Props> = ({ sortField, setSortField, todos }) => {
+  const oneTodoCompleted = todos.find(todo => todo.completed);
+
+  const activeTodos = todos.reduce(
+    (acc, todo) => (todo.completed ? acc : acc + 1),
+    0,
+  );
```

Comment:
You can use one iteration in reduce and return object with completed and uncompleted fields. ( not critical)